pkg/kubelet/dockershim/network/hostport/hostport.go

```diff
@@ -25,6 +25,8 @@ import (
 
 	"k8s.io/api/core/v1"
 	utiliptables "k8s.io/kubernetes/pkg/util/iptables"
+	
+	"github.com/nokia/sctp"
```

Comment:
I think you need to update the BUILD file in this directory to account for this import addition.  run 'hack/update-bazel.sh' and I think it'll do that for you.